src/lib/stateMappings/simple.ts

```diff
@@ -1,18 +1,33 @@
-import { UiState, StateMapping } from '../../types';
+import { UiState, IndexUiState, StateMapping } from '../../types';
+
+function indexStateWithoutConfigure(uiState: IndexUiState): IndexUiState {
```

Comment:
`getIndexStateWithoutConfigure` would better reflect the intent of this function.

Reply:
https://github.com/algolia/instantsearch.js/pull/4075/commits/0df9e8ca4ffebff2c4a47fef7d2813d5ee35911d